core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -9585,4 +9585,12 @@ private void checkJoinAssociateRuleWithTopAlwaysTrueCondition(boolean allowAlway
         .withRule(CoreRules.MULTI_JOIN_OPTIMIZE)
         .check();
   }
+
+  @Test void testIntersectToExistsRule() {
```

Comment:
1. Please add the Jira link
2. Add more test cases about 
-  INTERSECT ALL
-  INTERSECT on multiple fields
-  Multiple INTERSECT
3. We may not adopt this rule as the default one for now, but it can be set as the default rule to test whether the rule function is normal.

Reply:
Thanks for your reply, I will add jira link and more tests.